data/json/items/armor/jewelry.json

```diff
@@ -1322,8 +1312,7 @@
     "price_postapoc": 25,
     "material": [ "leather" ],
     "symbol": "-",
-    "color": "dark_gray",
-    "flags": [ "FANCY" ]
+    "color": "dark_gray"
```

Comment:
I'd give it FANCY and add a version with (blunt) spikes that would be likewise, personally.